### PR TITLE
feat: load sprites from git

### LINKED ISSUE #3: add pokemon sprite package

### PR DESCRIPTION
**Pull request by rdmulford**
idk why i didnt think of this before tbh

it loads basically instantly, try pulling down this branch and testing it out

related #18 
closes #3 
closes #18

### CODE REVIEW

File: src/components/PokemonList.tsx [hunk 1]
```diff
@@ -48,6 +51,11 @@ const PokemonList = () => {
                   bg="#d4d4d4"
                   borderColor="737373"
                   key={item.id}>
+                  <Image
+                    source={{uri: item.img}}
+                    style={{width: 90, height: 90}}
```
Comment (rdmulford) [inline src/components/PokemonList.tsx:56]: probably want to not use hardcoded size values in the future, for now its fine https://github.com/rdmulford/pokerus/issues/20

**Comment by mulfordkl** [comment]
Oh hella nice. Do those images get bundled with the app or fetched at runtime?

**Comment by rdmulford** [comment]
> Oh hella nice. Do those images get bundled with the app or fetched at runtime?

it gets fetched. Only concern i have is its possible we could hit github api rate limits. I think authenticated users get a much higher limit so we could add git creds. we'll also want to verify that content in our list is lazy loaded so its not making tons of requests every time.

i think at the very least there is some caching going on already because if you load the pokedex page, then go to Moves and then back to pokedex it loads way faster than first load

**Comment by rdmulford** [comment]
as discussed, we can add functionality to fetch asset data at app install and store in app storage, thus removing concerns around having lots of api calls. https://github.com/rdmulford/pokerus/issues/21

in addition, we can add a 'development' mode set by an ENV variable which turns off asset fetching and uses some placeholder asset instead https://github.com/rdmulford/pokerus/issues/22

**Pull request merged**